src/highlight/highlight.ts

```diff
@@ -1,4 +1,3 @@
-const get = require('lodash/get');
```

Comment:
This source file shouldn't have been touched.

Reply:
oops. my bad 0a5da47